packit/api.py

```diff
@@ -292,3 +293,80 @@ def create_srpm(self, output_file: str = None) -> Path:
                 up.bump_spec(version=version, changelog_entry="Development snapshot")
         srpm_path = up.create_srpm(srpm_path=output_file)
         return srpm_path
+
+    def status(self):
```

Comment:
Can't wait to make this work once we have packit service; imagine doing this on github:
```
/packit status
```

And packit adds a new comment:
```
Dist-git versions:
...

GitHub upstream releases:
...

Latest builds:
...
```

Reply:
In that case, this method should return a string containing status output.

Reply:
don't change it, we'll improve in future; I think it's just fine for now